packages/dashboard-v2/src/contexts/portal-settings/PortalSettingsContext.js

```diff
@@ -0,0 +1,9 @@
+import { createContext } from "react";
+
+export const defaultSettings = {
+  areAccountsEnabled: false,
+  isAuthenticationRequired: true,
+  isSubscriptionRequired: true,
```

Comment:
```suggestion
  areAccountsEnabled: false,
  isAuthenticationRequired: false,
  isSubscriptionRequired: false,
```

You probably want to turn them all false as default ?